es-vue-base/src/lib-components/EsVideo.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+    <div>
+        <b-embed
+            v-if="showVideo"
+            allowfullscreen
+            aspect="16by9"
+            :src="autoplayUrl"
+            :title="altText"
+            type="iframe" />
+        <es-button
+            v-else
+            class="VideoWithCover-button border-0 p-0 position-relative text-gray w-100"
+            @click="showVideo = true">
+            <b-img
```

Comment:
Since it would be ideal to still allow folks to use `<storyblok-image>` in the Nuxt repos for this, what do you think about using `<slot>` for the image here, with this `b-img` inside it as a fallback?
- see `EsSupport` for the example of this pattern
- probs remove the `sizes` attribute on the b-img, I think that's a nuxt-img prop
- the image url & alt props would become optional, and the fallback b-img would show only if those props are given & the slot is not